examples/ai-image-generator/src/assets/style.css

```diff
@@ -0,0 +1,69 @@
+@import "mirotone/dist/styles.css";
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+.grid-wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  gap: 10px; /* Optional: Add gap between elements */
+}
+
+body {
+  display: flex;
+}
+
+#root {
+  overflow: auto;
+  padding: var(--space-medium);
+}
+
+.button {
+  width: 300px;
+}
+
+/* Style for the container div */
+.image-container {
+  width: 300px; /* Make the container div as wide as its parent */
+  max-width: 100%; /* Ensure the container doesn't exceed its parent's width */
+  height: 300px; /* Start with a zero height */
+  position: relative; /* Establish a positioning context for child elements */
+  border: none;
+}
+
+/* Style for the image */
+.image-container img {
+  position: absolute; /* Position the image absolutely within the container */
+  width: 100%; /* Make the image as wide as the container */
+  height: 100%; /* Make the image as high as the container */
+  object-fit: cover; /* Ensure the image covers the entire container while maintaining its aspect ratio */
+  background: url(https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcRvF4WdZJSA4MkWJXClae4eCvSdk87c5Ok63fgxBxVyR6aHB2Ju_A)
+    no-repeat center;
+}
+
+.spinner {
+  position: fixed;
+  height: 2em;
+  width: 2em;
+  border: 3px solid rgba(255, 255, 255, 0.3);
```

Comment:
Can we use mirotone vars here?

Reply:
went through the Mirotone docs but not sure which ones you are referring to here? Which one specifically?

Reply:
added.